projects/storefrontlib/layout/main/storefront.component.ts

```diff
@@ -105,6 +115,18 @@ export class StorefrontComponent implements OnInit, OnDestroy {
         })
       );
     }
+
+    if (this.featureConfigService.isEnabled('a11yHamburgerMenuTrapFocus')) {
+      this.isExpanded$
```

Comment:
Lets put this in a protected function to reduce the length `ngOnInit` and so that we can name our function something that describes why we are subscribing.